Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -2187,15 +2187,29 @@ private void createCaseNodeData(ProgressIndicator progressIndicator) throws Case
     private void updateCaseNodeData(ProgressIndicator progressIndicator) throws CaseActionException {
```

Comment:
Could add param for ProgressIndicator  to the header.

Reply:
@raman-bt, thanks I definitely need to do that.